updater/spec/dependabot/updater/error_handler_spec.rb

```diff
@@ -232,10 +236,6 @@
         end
       end
 
-      before do
-        allow(Dependabot::Experiments).to receive(:enabled?).with(:record_update_job_unknown_error).and_return(false)
```

Comment:
These explicit mocks were not needed once we stop mocking (default to false), so I deleted them.